gpiozero/boards.py

```diff
@@ -1053,6 +1053,71 @@ def __init__(self, pwm=False, initial_value=False, pin_factory=None):
         )
 
 
+class RGBLEDBoard(CompositeOutputDevice):
+    """
+    Extends :class:`CompositeOutputDevice`. and represents a generic RGBLED
+    board or collection of LEDs.
+
+    The following example turns on all the LEDs on a board containing 5 LEDs
+    attached to GPIO pins 2 through 6::
+
+        from gpiozero import RGBLEDBoard
+
+        leds = RGBLEDBoard((2, 3, 4), (5, 6, 7), (7, 8, 9))
```

Comment:
Hmm, probably shouldn't be using GPIO7 twice?

Reply:
whoops!